src/Form/ImportVideoItemByMpxId.php

```diff
@@ -98,13 +102,22 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
 
     $request = new UpdateVideoItemRequest($mpx_id, $video_type);
     try {
-      $this->updateVideoItem->execute($request);
-      $this->messenger()->addMessage($this->t('The selected video has been imported.'));
+      $response = $this->updateVideoItem->execute($request);
+      if (empty($response->getUpdatedEntities())) {
+        $mpx_media = $response->getMpxItem();
+        $this->messenger()->addWarning($this->t("The selected video: @video_title (@id) did not import. The video was filtered out by one or more custom import filters. Adjust the video metadata in mpx to ensure it's available to be imported and try again.", [
```

Comment:
Would this message be shown on screen? If so, I wonder if it will be correct for an editor.

Reply:
Yes, it will be shown on screen. Editors definitely wouldn't know what an import filter is. 🤔 The most we can tell at this point is that the video was filtered out. We don't know why yet. There is some discussion about how we could eventually be more specific about why here: https://github.com/Lullabot/media_mpx/issues/142. Here is some alternate language off the top of my head, any of these sound better?

> The selected video: @video_title (@id) did not import. Adjust the video metadata in mpx to ensure it's available to be imported and try again.

> The selected video: @video_title (@id) did not import. One or more custom business rules are in place which filtered it out (for example Unapproved videos may be filtered out). Adjust the video metadata in mpx to ensure it's available to be imported and try again.

Shout out the site administrator:

> The selected video: @video_title (@id) did not import. There may be one or more custom business rules in place which filtered it out. Consult the site administrator and adjust the video metadata in mpx to ensure it's available to be imported and try again.

Reply:
I think that the latter would be the most clear to me for the short term. As for the long term, #142 would make this crystal clear.